### PR TITLE
Removed references to deleted architectures

### PR DESCRIPTION
**Pull request by lmondada**

### CODE REVIEW

**Comment by lmondada** [comment]
Is there any formatting conventions I should respect for the `ipynb` file or are the notebooks run on the CI anyway?

**Review by cqc-alec — CHANGES_REQUESTED** [review]
These ipynb files are now generated so shouldn't be edited manually. Instead, you should edit the script in the `python` directory and regenerate the ipynb file from there (see the instructions in the README). Thanks!

**Comment by lmondada** [comment]
> These ipynb files are now generated so shouldn't be edited manually. Instead, you should edit the script in the `python` directory and regenerate the ipynb file from there (see the instructions in the README). Thanks!

Apologies. I hadn't seen that. Thanks for the pointer

File: examples/python/routing_example.py [hunk 1]
```diff
@@ -72,21 +72,9 @@ def draw_graph(coupling_map):
...
-
-# and the CyclicButterfly:
-
-from pytket.routing import CyclicButterfly
-
-cb_architecture = CyclicButterfly(3)
-print(cb_architecture)
-draw_graph(cb_architecture.coupling)
+# In many cases, we are interested in the architectures of real devices.
+# These are available directly from the device backends, available within tket's respective extension packages.
```
Comment (cqc-alec) [inline examples/python/routing_example.py:76]: Unfortunately `p2j` isn't clever enough to render these three lines as one paragraph (it inserts line breaks), so I think they need to be all on one line.

**Comment by cqc-alec** [comment]
> Apologies. I hadn't seen that. Thanks for the pointer

No problem, it is a new change made the other day :-)

File: examples/python/routing_example.py (hunk 1 above)
Comment (lmondada) [reply examples/python/routing_example.py]: done

**Review by cqc-alec — APPROVED** [review]

**Pull request merged**